packages/react-components/react-tabs/src/components/Tab/useTabStyles.ts

```diff
@@ -43,23 +37,23 @@ const useRootStyles = makeStyles({
     textTransform: 'none',
   },
   mediumHorizontal: {
-    columnGap: pendingSpacingTokens.sNudge,
+    columnGap: tokens.spacingHorizontalSNudge,
     justifyContent: 'center',
-    ...shorthands.padding(pendingSpacingTokens.m, pendingSpacingTokens.mNudge),
+    ...shorthands.padding(tokens.spacingVerticalM, tokens.spacingHorizontalMNudge),
   },
   mediumVertical: {
-    columnGap: pendingSpacingTokens.sNudge,
+    columnGap: tokens.spacingHorizontalSNudge,
```

Comment:
```suggestion
    columnGap: tokens.spacingVerticalSNudge,
```

There are perhaps more places where vertical should be used